cdap-docs/admin-manual/source/security/impersonation.rst

```diff
@@ -47,7 +47,3 @@ The configured HDFS delegation token timeout must be longer than the configured
 partition duration (``stream.partition.duration``), which has a default value of
 one hour (3600000). It must also be larger than the log saver's maximum file
 lifetime (``log.saver.max.file.lifetime.ms``), which has a value of six hours (21600000).
-
```

Comment:
Should we then change the note to instead say that per 8140, if you have Hive 0.13 and impersonation, cdap explore does not work?